### PR TITLE
Set useGOMEMLIMIT to true for testing

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** 

- Test setting `useGOMEMLIMIT: true` for testing open-telemetry/opentelemetry-helm-charts/pull/900
- Change base image from xenial to focal, since the xenial image is no longer available

### CODE REVIEW

File: ci/values-staging.yaml [hunk 1]
```diff
@@ -3,3 +3,7 @@ nodeSelector:
 presets:
   logsCollection:
     enabled: false
+useGOMEMLIMIT: true
+resources:
+  limits:
+    memory: 160Mi
```
Comment (mackjmr) [inline ci/values-staging.yaml]: Should we remove this, as this was just for testing purposes ?

File: ci/values.yaml [hunk 2]
```diff
@@ -66,6 +66,7 @@ config:
         http:
           endpoint: 0.0.0.0:4318
   exporters:
+    debug: null
```
Comment (mackjmr) [inline ci/values.yaml:70]: Could you add a comment on when we can remove this (I think once we merge upstream should be good ?)

File: ci/values-staging.yaml (hunk 1 above)
Comment (mackjmr) [reply ci/values-staging.yaml]: the `memory: 160Mi`, not the `useGOMEMLIMIT: true`

**Review by songy23 — APPROVED** [review]

File: ci/values-staging.yaml (hunk 1 above)
Comment (mx-psi) [reply ci/values-staging.yaml]: Removed!

File: ci/values.yaml (hunk 2 above)
Comment (mx-psi) [reply ci/values.yaml:70]: Done

**Review by mackjmr — APPROVED** [review]

**Pull request merged**